README.md

```diff
@@ -66,6 +66,40 @@ Assisted digital satisfaction surveys:
 * https://www.gov.uk/bank-holidays
 * https://www.gov.uk/when-do-the-clocks-change
 
+### Misc
+
+| URL                                                                                                                                                          |  | Related Views             |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------ |  | ------------------------- |
```

Comment:
This table isn't rendering as a table, just as the raw markdown; I think this is because of the whitespace between two pipes on line 72.

I think that this should be two columns to make it easier to navigate, unless there was some missing content in the middle column?

```suggestion
| URL | Related Views |
| - | - | 
```

Reply:
🙈  I've adjusted this entirely - this [previewer](https://markdownlivepreview.com/) seemingly doesn't use the same renderer.

Reply:
Argh, how annoying! Looks good now though 👍